modules/developer-lightspeed/con-rag-embeddings.adoc

```diff
@@ -3,4 +3,6 @@
 [id="con-rag-embeddings_{context}"]
 = Retrieval Augmented Generation embeddings
 
-The {product} documentation set has been added to the {rcs-name} as a RAG embedding.
+The {product} documentation set has been added as a Retrieval-Augmented Generation (RAG) data source.
+
+The {lcs-name} ({lcs-short}) sidecar handles the RAG process, using specialized Llama Stack components to manage the documentation data and generate vector embeddings.
```

Comment:
```suggestion
The RAG embeddings in the vector database is used by the Llama Stack service as tool to provide references to production documentation during the inference process.
```

We can talk about how

1. RAG is initialized as an init container and teh data copied to a volume
2. Llama stack sidecar mounts RAG data from above volume

All of this is in the snippet here https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1467/files#r2479399288